### PR TITLE
Added color palette.

### PR DESCRIPTION
**Pull request by hog-seruj**

### CODE REVIEW

File: color/preview.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * @file
+ * This is simple js for management of color module display.
+ */
+
+(function ($, Drupal, drupalSettings) {
```
Comment (iberdinsky-skilld) [inline color/preview.js]: seems drupalSettings not needed here. same as jQuery imho.

**Comment by iberdinsky-skilld** [comment]
also https://www.npmjs.com/package/css-vars-ponyfill should be added i guess.

File: color/preview.js (hunk 1 above)
Comment (hog-seruj) [reply color/preview.js]: Fixed.

File: kaizen.info.yml [hunk 2]
```diff
@@ -7,3 +21,16 @@ regions:
   header: Header
   content: Content
   footer: Footer
+
+# The "libraries:" section will add a library to _all_ pages.
+libraries:
+  - kaizen/base
+
+# This section is used by the contrib module, Component Libraries. It allows you
```
[the remark this answers is not in the document]
Comment (hog-seruj) [reply kaizen.info.yml]: Removed.

File: color/preview.js [hunk 3]
```diff
@@ -3,17 +3,17 @@
  * This is simple js for management of color module display.
  */
 
-(function ($, Drupal, drupalSettings) {
+(function (Drupal) {
   Drupal.color = {
     logoChanged: false,
     callback: function callback(context, settings, $form) {
+      var $colorPreview = $form[0].querySelectorAll('.color-preview .brands__item > div:first-of-type');
```
[the remark this answers is not in the document]
Comment (hog-seruj) [reply color/preview.js]: This code isn't in ES6. I think var is ok here.

File: kaizen.info.yml [hunk 4]
```diff
@@ -7,3 +21,14 @@ regions:
...
+
+# The "libraries:" section will add a library to _all_ pages.
+
+# This section is used by the contrib module, Component Libraries. It allows you
+# to reference .twig files in your sass/ directory by using the Twig namespace:
+# @kaizen_theme
+component-libraries:
+  kaizen:
+    paths:
+      - src/sass
```
Comment (andypost) [inline kaizen.info.yml]: Why sass used here? No need expose implementation details thid way

File: color/preview.css [hunk 5]
```diff
@@ -0,0 +1,17 @@
+.brands {
```
Comment (andypost) [inline color/preview.css]: why `brands` used, I think `.color` & `.color_item` better

File: color/preview.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
+<div class="color-preview">
+  <div id="text">
+    <h2>Color Palette</h2>
+    <div class="brands">
```
Comment (andypost) [inline color/preview.html]: I think `class="color"` is better choice

Comment (hog-seruj) [reply color/preview.html]: It's not necessary. We use $brand variable name

File: kaizen.info.yml [hunk 7]
```diff
@@ -7,3 +21,14 @@ regions:
   header: Header
   content: Content
   footer: Footer
+
+# The "libraries:" section will add a library to _all_ pages.
+
+# This section is used by the contrib module, Component Libraries. It allows you
+# to reference .twig files in your sass/ directory by using the Twig namespace:
+# @kaizen_theme
+component-libraries:
```
Comment (iberdinsky-skilld) [inline kaizen.info.yml]: still here

File: src/js/init.js [hunk 8]
```diff
@@ -1 +1,4 @@
 import './.modernizrrc';
+import cssVars from 'css-vars-ponyfill';
```
Comment (iberdinsky-skilld) [inline src/js/init.js]: maybe add modernizr check?

Comment (iberdinsky-skilld) [reply src/js/init.js]: should be something like
`
Modernizr.on('customProperties', function( result ) {
  if (result) {
   // the browser has cssVars
  } else {
    // CSSVARS POLIFILL
  }
})
`

Comment (hog-seruj) [reply src/js/init.js]: Modernizer added.

**Comment by iberdinsky-skilld** [comment]
lgtm

**Pull request merged**